server/database/data/sets.json

```diff
@@ -0,0 +1,53 @@
+[
+  {
+    "name": "Sinistrofu Set",
+    "items": ["Sinistrofu Cloak", "Sinistrofu Amulet", "Sinistrofu Boots"],
+    "bonuses": [
+      {
+        "item_count": 2,
```

Comment:
camelCase

Reply:
Also, it seems like it makes more sense for bonuses to be an object like
```
"bonuses: {
  "2": [
    {"stat": "Chance", ...}
    ]
}
```
where the key is the number of items, so it's easier to access the set bonus that the UI will be looking for